packages/react-ui/internal/Popup/Popup.tsx

```diff
@@ -90,6 +90,7 @@ export interface PopupProps extends CommonProps, PopupHandlerProps {
   positions: PopupPosition[];
   useWrapper: boolean;
   ignoreHover: boolean;
+  menuWidth: string | number;
```

Comment:
Для попапа это просто ширина - `width`. Сущность menu - это абстракция конкретного использования.
И здесь так же можно использовать `CSSProperties`:
```suggestion
  width: React.CSSProperties['width'];
```